apps/emqx/src/emqx_schema.erl

```diff
@@ -1093,13 +1095,42 @@ to_bar_separated_list(Str) ->
 to_ip_port(Str) ->
     case string:tokens(Str, ":") of
         [Ip, Port] ->
-            case inet:parse_address(Ip) of
+            case get_addr(Ip) of
                 {ok, R} -> {ok, {R, list_to_integer(Port)}};
                 _ -> {error, Str}
             end;
         _ -> {error, Str}
     end.
 
+get_addr({_1, _2, _3, _4} = IPV4) ->
```

Comment:
It seems that this part of the code will not be run.

Reply:
it, ok,  and run correctly